cmake/Modules/winconf.cmake

```diff
@@ -1,102 +1,26 @@
 # Settings for compiling qBittorrent on Windows
 
-list(APPEND CMAKE_LIBRARY_PATH "$ENV{LIB}")
-
-set(LibtorrentRasterbar_CUSTOM_DEFINITIONS
-    -DBOOST_ASIO_DISABLE_CONNECTEX
-    -DBOOST_EXCEPTION_DISABLE
-    -DTORRENT_USE_LIBCRYPTO
-    # TODO: remove the following define as it is not used since OpenSSL >= 1.1
-    -DTORRENT_USE_OPENSSL
-    # TODO: remove the following define as it is not used since libtorrent >= 1.2
-    -DTORRENT_DISABLE_RESOLVE_COUNTRIES
+add_compile_definitions(
+    NTDDI_VERSION=0x06010000
```

Comment:
all these defined will be added automatically

Reply:
@jagannatharjun 
Are you sure? My impression is that unless you specify `NTDDI_VERSION`, `_WIN32_WINNT` and `_WIN32_IE` manually with the appropriate version, then it is not guaranteed that the executable will work all the way back to Windows 7. But I may be wrong or there may be a better way to do this. Do you happen to know how?

Regarding the other definitions, how can I check if they are being added automatically without trial-and-error?

Reply:
`UNICODE` may be helpful, but I don't really you think we need other flags anyway

Reply:
@jagannatharjun 
After reading up on it some more, looks like at least `WIN32` (automatically added by CMake) and `_WIN32` (automatically added by MSVC compiler - https://docs.microsoft.com/en-us/cpp/preprocessor/predefined-macros?view=vs-2019) are redundant, so I removed them.